packages/visualizations/src/components/Map/types.ts

```diff
@@ -37,11 +37,11 @@ export interface MapFilter {
     value: string | string[];
 }
 
-export interface GeoJsonChoroplethOptions extends ChoroplethOptions {
+export interface ChoroplethGeoJsonOptions extends ChoroplethOptions {
```

Comment:
One of the "easy" benefit of splitting IMO would be to not require this `shapes` object from the outside.

We could have directly `shapes` expecting a `GeoJSON` here, and for `ChoroplethVectorTilesOptions` directly have a `shapesTiles` property (a different name for clarity) expecting `ChoroplethShapeVectorTilesValue` but without the `type` anymore.

Then `ChoroplethGeoJson.svelte` and its VTiles counterpart have already the code that handles it, so it shouldn't make things too different. From the outside's point of view, using each of these components will be much easier to understand.